eZ/Publish/Core/MVC/Symfony/Security/EventListener/SecurityListener.php

```diff
@@ -213,6 +214,15 @@ public function onKernelRequest( GetResponseEvent $event )
             return;
         }
 
+        try
+        {
+            $this->repository->getUserService()->loadUser( $this->repository->getCurrentUser()->contentInfo->id );
+        }
+        catch ( NotFoundException $e )
+        {
+            $this->repository->setCurrentUser( $this->repository->getUserService()->loadUser( 10 ) );
```

Comment:
ok. my doubt is, if the user has been deleted what ``` $this->repository->getCurrentUser()``` returns? something from the session?